cadcdata/cadcdata/tests/test_core.py

```diff
@@ -323,8 +321,19 @@ def test_help(self):
   -u, --user USER       Name of user to authenticate. Note: application
                         prompts for the corresponding password!
   -v, --verbose         verbose messages
-  -V, --version         show program's version number and exit
   --wcs                 Return the World Coordinate System (WCS) information
+
+Examples:
+- Anonymously getting a public file:
+        cadc-data get -v -a GEMINI 00aug02_002.fits
```

Comment:
There are archives that store fits files as  NAME.fz instead of NAME.fits.fz  .. hmm.